ui/packages/atlasmap-core/src/models/expression.model.ts

```diff
@@ -93,7 +93,16 @@ export class FieldNode extends ExpressionNode {
           )!;
         }
         if (!this.mappedField) {
-          this.mappedField = mapping.getReferenceField(fieldParts[1]);
+          const docDef = ConfigModel.getConfig().getDocForName(
```

Comment:
The field expression is `<Document ID>:<Field path>`, not the document name I think

Reply:
Ah - you're right - can use fieldParts[0] as is since it's already docId str